.travis.yml

```diff
@@ -1,44 +0,0 @@
-language: node_js
-node_js:
-  - 'lts/*'
-
-branches:
-  only:
-    - develop
-os:
-  - linux
-dist: xenial
-
-before_install:
-  - npm config set '//npm.pkg.github.com/:_authToken' $GH_TOKEN
-  - npm config set @typeform:registry https://npm.pkg.github.com/
-
-install:
-  - npm ci
-
-script:
-  - npm run build
-  - travis_retry npm run travis
-
-env:
-  global:
-    - PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=1
-  jobs:
-    - BROWSER=mac-chrome-latest
-    - BROWSER=mac-firefox-latest
-    - BROWSER=mac-safari-latest
-    - BROWSER=windows-chrome-latest
-    - BROWSER=windows-firefox-latest
-    - BROWSER=windows-edge-latest
-    - BROWSER=ios-latest
-    # - BROWSER=android-latest
-
-jobs:
-  allow_failures:
-    - env: BROWSER=android-latest
```

Comment:
I've removed `android-latest` browser since it was always failing and here was added under `allow_failures`